browser.js

```diff
@@ -26,13 +32,30 @@ function randomBytes (size, cb) {
   }
 
   // XXX: phantomjs doesn't like a buffer being passed here
-  var bytes = Buffer.from(rawBytes.buffer)
+  return Buffer.from(rawBytes.buffer)
+}
+
+function randomBytes (size, cb) {
+  if (size > MAX_UINT32) {
+    throw new Error('Requested too many random bytes - the maximum allowed are ' + MAX_UINT32)
+  }
+
+  var bytes = Buffer.alloc(0)
+
+  if (size > MAX_BYTES) {
+    bytes = randomBytes(size - MAX_BYTES)
+
+    bytes = Buffer.concat([bytes, generateBytes(MAX_BYTES)])
+  } else if (size > 0) {
```

Comment:
how about if size is less then or equal to zero then we make an empty buffer

Reply:
Node throws an error if `size` is less than zero.  I've added this check to this PR.